hypervisor/src/kvm/mod.rs

```diff
@@ -66,14 +66,16 @@ pub use {
     kvm_bindings::kvm_vcpu_events as VcpuEvents, kvm_ioctls::DeviceFd, kvm_ioctls::IoEventAddress,
     kvm_ioctls::VcpuExit,
 };
-#[derive(Copy, Clone, Default)]
-pub struct VmState {}
+#[derive(Clone, Copy, Debug, PartialEq, Deserialize, Serialize)]
+pub struct KvmVmState {}
```

Comment:
Could you please fix all this in the first commit directly?
Applies for the entire commit here.